src/Microsoft.ML.Data/DataLoadSave/Binary/BinaryLoader.cs

```diff
@@ -757,7 +730,7 @@ public void GetMetadata<TValue>(string kind, int col, ref TValue value)
         /// </summary>
         private const ulong ReaderFirstVersion = 0x0001000100010002;
 
-        public Schema Schema { get { return _schema; } }
+        public Schema Schema { get { return _outputSchema; } }
```

Comment:
>public Schema Schema { get { return _outputSchema; } } [](start = 8, length = 54)

Cool thanks @wshin. If perchance you happen to need to do some more commits, no harm C# 6/7-ifying these properties. But no hard feelings if you don't of course, just if quite convenient.